### PR TITLE
NO-JIRA: container log JSON schema - minor description update

### PR DESCRIPTION
**Pull request by tremes**
This PR implements a new data enhancement to...

## Categories

- [ ] Bugfix
- [X] Data Enhancement
- [ ] Feature
- [ ] Backporting
- [ ] Others (CI, Infrastructure, Documentation)

## Sample Archive

- `path/to/sample_data.json`

## Documentation

- `path/to/documentation.md`

## Unit Tests

- `path/to/file_test.go`

## Privacy

Yes. There are no sensitive data in the newly collected information.

## Breaking Changes

Yes/No

## References

https://issues.redhat.com/browse/???
https://access.redhat.com/solutions/???

### CODE REVIEW

File: pkg/gatherers/conditional/container_log.schema.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
             "uniqueItems": false,
             "items": {
                 "type": "string",
-                "description": "Regular expression used to filter the container log"
+                "description": "Regular expression used to filter the container log. The Go's Match (https://pkg.go.dev/regexp#Regexp.Match) function is used, so if a log line has a match, it is included in the collected data."
```
Comment (jholecek-rh) [inline pkg/gatherers/conditional/container_log.schema.json:37]: The wording is developer-centric a bit more than I would expect in this context. :) More importantly, the causality between using the Go's Match function and the rest of the sentence doesn't feel right.

How about this?

```suggestion
                "description": "Regular expression used to filter the container log. The collected data will include all lines matching the regular expression (as implemented by https://pkg.go.dev/regexp#Regexp.Match)."
```

Comment (tremes) [reply pkg/gatherers/conditional/container_log.schema.json:37]: OK :) Thanks

**Comment by tremes** [comment]
This minor update not affecting any functionality
/override ci/prow/insights-operator-e2e-tests 
/override ci/prow/e2e-metal-ipi-ovn-ipv6 
/override ci/prow/e2e-agnostic-upgrade 
/override ci/prow/e2e

**Review by ncaak — APPROVED** [review]
/lgtm

**Pull request merged**